components/style/properties/longhand/position.mako.rs

```diff
@@ -220,3 +220,15 @@ ${helpers.single_keyword("object-fit", "fill contain cover none scale-down",
                               spec="https://drafts.csswg.org/css-grid/#propdef-%s" % longhand,
                               products="gecko")}
 % endfor
+
+${helpers.predefined_type("grid-row-gap",
+                          "LengthOrPercentage",
```

Comment:
You should add a `products=gecko` here (and below). We don't have grid support in Servo right now.